setup/azurecmdfiles.wxi

```diff
@@ -578,6 +578,15 @@
                         <Component Id="cmpF08FE4050A937C6DD895E92ABAE2A275" Guid="*">
                             <File Id="fil7EAD0480A7385B639C6A89FD485C13D0" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureBatch\Microsoft.Azure.ResourceManager.dll" />
                         </Component>
+                        <Component Id="cmp1D1A78885A81C3FE9B8F9FBFAD6EB010" Guid="*">
```

Comment:
this is too many changes in the wix file - the only change that should be occuring is the additon of your ps1xml file.  Please clean your directory, rerun, and regenerate the msi.